src/Aspire.Dashboard/Model/BrowserLinkOutgoingPeerResolver.cs

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using Aspire.Dashboard.Otlp.Model;
+
+namespace Aspire.Dashboard.Model;
+
+public sealed class BrowserLinkOutgoingPeerResolver : IOutgoingPeerResolver
+{
+    public IDisposable OnPeerChanges(Func<Task> callback)
+    {
+        return new NullSubscription();
+    }
+
+    private sealed class NullSubscription : IDisposable
+    {
+        public void Dispose()
+        {
+        }
+    }
+
+    public bool TryResolvePeerName(OtlpSpan span, [NotNullWhen(true)] out string? name)
+    {
+        // There isn't a good way to identify the HTTP request the BrowserLink middleware makes to
+        // the IDE to get the script tag. The logic below looks at the host and URL and identifies
+        // the HTTP request by its shape.
+        // Example URL: http://localhost:59267/6eed7c2dedc14419901b813e8fe87a86/getScriptTag
+        //
+        // There is the chance future BrowserLink changes make this detection invalid.
+        // Also, it's possible to misidentify a HTTP request.
+        //
+        // A long term improvement here is to add tags to the BrowserLink client and then detect the
+        // values in the span's attributes.
+        var url = OtlpHelpers.GetValue(span.Attributes, "http.url");
+        if (url != null && Uri.TryCreate(url, UriKind.Absolute, out var uri))
+        {
+            if (string.Equals(uri.Host, "localhost", StringComparison.OrdinalIgnoreCase))
+            {
+                var parts = uri.AbsolutePath.Split('/', StringSplitOptions.RemoveEmptyEntries);
+                if (parts.Length == 2)
+                {
+                    if (Guid.TryParse(parts[0], out _) && string.Equals(parts[1], "getScriptTag", StringComparison.OrdinalIgnoreCase))
+                    {
+                        name = "browserlink";
```

Comment:
Marketing would probably be annoyed if we called it anything other than "Browser Link" in text (I suppose same goes for the comments, but that's less important than what's visible to the user).

Reply:
😆 

It can be whatever we want it to be. I chose `browserlink` as the other resource names are one word and lowercase. `Browser Link` would be inconsistent.

Reply:
Yeah, agree it'd be inconsistent in this context. That was just one of my first "marketing" lessons I learned when I joined WebTools so I have a hard time not calling it out 🤣

Reply:
Changed it to Browser Link. See if anyone in #TeamConsistency cares enough to bring it up.